.travis.yml

```diff
@@ -16,12 +17,6 @@ env:
     # See https://git.io/vdao3 for details.
     - JOBS=1
 
-before_install:
```

Comment:
why did this get ditched?

Reply:
Node 8 has NPM 6 by default so these install steps and the spin workaround are no longer necessary. I should have removed it when I dropped node 6 support, but the ember blueprint just got updated as they dropped node 6.

Reply:
thanks!